File: dixit-companion/src/main/java/fr/erban/dxitcompanion/db/player/PlayerEntity.java
package fr.erban.dxitcompanion.db.player;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class PlayerEntity {

    @PrimaryKey
    @NonNull
    private String name;

    private int nbGames;

    private int nbWins;
}

File: dixit-companion/src/main/java/fr/erban/dxitcompanion/db/player/PlayerEntity.kt
package fr.erban.dxitcompanion.db.player

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class PlayerEntity(
    @PrimaryKey
    val name: String,
    val nbGames: Int = 0,
    val nbWins: Int = 0,
    val colorHex: String = "#5C3A9E",
    val emoji: String = "🎭"
)
